internal/action/command.go

```diff
@@ -439,21 +439,35 @@ func (h *BufPane) openHelp(page string) error {
 
 		if h.Buf.Type == buffer.BTHelp {
 			h.OpenBuffer(helpBuffer)
-		} else {
+		} else if hsplit {
 			h.HSplitBuf(helpBuffer)
+		} else {
+			h.VSplitBuf(helpBuffer)
 		}
 	}
 	return nil
 }
 
 // HelpCmd tries to open the given help page in a horizontal split
 func (h *BufPane) HelpCmd(args []string) {
+	hsplit := true
 	if len(args) < 1 {
 		// Open the default help if the user just typed "> help"
-		h.openHelp("help")
+		h.openHelp("help", hsplit)
 	} else {
+		if args[0] == "hsplit" || args[0] == "vsplit" {
```

Comment:
It might be better to add a flag e.g. `-v` (meaning vsplit) that can be anywhere in the command, not necessarily the 1st argument, similarly to the `replace` command.

...Or maybe we shouldn't overthink this and should simply add an option, e.g. `helpsplit`.

Reply:
I thought using `(h|v)split` is more intuitive and allows easier completion. But ok, since @Andriamanitra seems to agree with you we go with the flag-approach.
Furthermore I prefer the flag over a new help-command...at least for now. But it doesn't prevent it, since we can simply add it similar to the `replaceall`.

PS:
Adding flag-completion would be a nice thing. :thinking:

Reply:
The reason I would prefer `-flag` syntax is that I would expect `help hsplit` to bring up the help for the hsplit command. `help -hsplit` makes the role of `-hsplit` more clear.

Reply:
Hm, it's the other way around.
Any preferences if we simply use `-v` as flag for `vsplit` instead of the default `hsplit`, or shall we use the more precise `-vsplit` and `-hsplit`?
`-v` is commonly used for verbosity.

Reply:
> Furthermore I prefer the flag over a new help-command

I didn't mean a new command, I meant a new option, which would configure whether the `help` command (without any extra flags) will open a horizontal or vertical split.

Arguably that would be more convenient to the user, since if the user prefer vertical splits, the user can just set this `helpsplit` option to `vsplit` once, and from then on enjoy using the `help` command without bothering with extra params.

Reply:
Ok, fair enough for simplicity. What about combining both worlds to give the user the flexibility to chose? :wink: